test/run_test.sh

```diff
@@ -203,6 +208,9 @@ function link_latest() {
 # If an outcome file was produced, keep a link to the latest file
 link_latest outcome json
 
+# If a traffic model file was produced, keep a link to the latest file
+link_latest traffic.model json
```

Comment:
I'm not sure.... The state transitions are probabilistic, so the results will be close but not identical after each run. I guess it would still be nice to output the diff lines for human comparison.